omnibus/config/software/chef.rb

```diff
@@ -44,6 +44,10 @@
 dependency "chef-gem-byebug"
 dependency "chef-gem-debug_inspector"
 dependency "chef-gem-binding_of_caller"
+unless ios_xr? || solaris?
```

Comment:
This is how lamont and thom recommended we do it, and the builds succeeded, so unless there's a specific reason we should I'm inclined to keep it.